README.md

```diff
@@ -1 +1,55 @@
-# Leaderboad
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Leaderboard:
+
+## Live Demo
```

Comment:
- [ ] Hi @ridwanediallo, this is not the correct link to your project. Please let's use the proper link here.